example-plugin/src/main/resources/register_fields.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Is there a reason this file is a main resource and not a test resource?

Reply:
++ good catch, moved it to test